internal/polling/pollster.go

```diff
@@ -0,0 +1,85 @@
+package polling
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"gophermart/internal/model"
+	"log/slog"
+	"sync"
+	"syscall"
+	"time"
+)
+
+// Нужно опрашивать внешний сервис Acrual с каким-то интервалом до тех пор, пока он не вернет нужный статус по заказу PROCESSED или INVALID
+
+type Pollster struct {
+	incoming    chan int
+	orders      []int
+	stopCh      chan struct{}
+	accrualAddr string
+	store       Store
+}
+
+func NewPollster(accrualAddr string, store Store) *Pollster {
+	incoming := make(chan int)
+	orders := make([]int, 0)
+	stopCh := make(chan struct{})
+	return &Pollster{
+		incoming,
+		orders,
+		stopCh,
+		accrualAddr,
+		store,
+	}
+}
+
+func (p *Pollster) Push(OrderID int) {
+	go func() {
+		p.incoming <- OrderID
+	}()
+}
+
+func (p *Pollster) Run() {
+	ticker := time.NewTicker(2 * time.Second)
+	for {
+		select {
+		case <-ticker.C:
+			taskNumber := len(p.orders)
```

Comment:
Не подумай, что я придираюсь 😊  но у меня `number` ассоциируется с номером, а не c количеством. 
```suggestion
			tasksCount := len(p.orders)
```